### ISSUE TITLE
Import/Export Account File

### ISSUE THREAD

**Issue by trymeouteh** [oxen-io/session-android#436]
Jami allows you to import/export your Jami ID to a file which can be import again and as long as you do not lose your file or forget the password to unlock the file you can always recover your Jami ID.

Session could have a file import/export for its user accounts as an alternative to writing down a word phrase. This file could be a json, 7z, gz, etc. I would highly recommended this file require a password which will encrypt the file. And this import/export feature could be added to the app onboarding process.

**Comment by Sepero** [comment]
I agree with this, but make password optional. Users should always be allowed to choose their own level of security.
